data/core/tips.json

```
@@ -503,5 +503,125 @@
+    "category": "tip",
+    "text": [ "Verticallity is a thing.  If you can get to the roof somehow, it may save your life." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Your enemy is often slower then you.  Walk to conserve stamina and run only when you must." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Fight on your conditions whenever you can.  Use terrain, traps and fire to your advantage." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Pick your fights and don't let them pick you.  Don't bite what you can't chew." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Learn to use zone sorting to sort your loot in one go.  Y to set up zones, O to sort items." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Want to loot everything not nailed down in one go?  Use zone manager to fill your hoard." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You can view local keybindings in most sub-menus by pressing a question mark (?) key." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "There are some unbound keybindings that might prove useful for your needs.  Check them out." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Check what you can craft from time to time, even if you don't have components right now." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Where applicable, use blacklists, whitelists and filters for a better quality of life." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Many items have contextual use.  Use item action menu to find out what you can do with them." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Action menu can help you determine what you can do.  There are also some rare actions there." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Auto-pickup settings can save your time when you need it.  It's fully customizable too." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Feeling unwell?  Allow yourself to rest, even for a few days as needed.  Heal now, explore later." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Overconfidence is a treacherous killer.  Leave yourself some leeway for unforeseen circumstances." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Be mindful of what, when and how much you eat.  A balanced diet is crucial in the long run." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Skillful management of your stamina is crutial for your survival.  Conserve it when you can." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You can make a crafting spot, mark a firewood source or mark a practice target with no cost." ]
```

Maybe add a mention that you can do it through construction menu?
```suggestion
    "text": [ "You can make a crafting spot, mark a firewood source or mark a practice target with no cost through the construction menu." ]
```

---

This one would too long (width limit), but I'll try to reword that.